sal_interfaces/Watcher/Watcher_Events.xml

```diff
@@ -57,15 +57,18 @@
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>escalated</EFDB_Name>
-      <Description>Has this alarm been escalated to backup personnel?</Description>
-      <IDL_Type>boolean</IDL_Type>
+      <EFDB_Name>escalatedId</EFDB_Name>
+      <Description>Blank if escalated not attempted.
```

Comment:
Shouldn't `escalated` be `escalate` or perhaps `escalation` here?

Reply:
Good catch. Changed to "escalation"